src/settings/spacePortalsSettings.ts

```diff
@@ -110,7 +110,7 @@ export const SPACE_INFO_LIST: SpaceInfoListItem[] = [
     poster: PosterEmbeddable,
     description: "",
     template: VenueTemplate.embeddable,
-    hidden: false,
+    hidden: true,
```

Comment:
Here we hide "embeddable" portals which do not require a space until we decide what to do with them. Were we going to get rid of them?